Prace_domowe/Praca_domowa4/Adrian_Stańdo/pd04.ipynb

```diff
@@ -0,0 +1,1202 @@
+{
```

Comment:
<p>Ogólnie jest git.</p><p>Fajnie, że jest przykład, który pokazuje, że skalowanie działa.</p><p>Mogłoby być więcej komentarzy.</p><p>Tylko to optymalizowanie degree to takie faux-paux</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/249/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>